website/developers/code/git-more.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Get the Code
+layout: website-normal
+menu_proxy_for: index.md
+children:
+- { link: index.html, title: Basics }
+- { section: Set Up Forks }
+- { section: Multi-Project Changes }
+- { section: How We Use Branches, title: Branches }
+- { section: About Submodules, title: Submodules }
+- { section: Not Using Submodules }
+- { section: Useful Aliases and Commands, title: Git Aliases }
+- { section: Legacy Incubator Pull Requests, title: Incubator PRs }
+---
+
+## Set Up Forks
+
+If you're contributing or working a lot on a feature, 
+you'll probably want your own forks and a slightly different git remote setup.
+
+You can create forks of each Brooklyn repo [in the GitHub UI](https://github.com/apache?query=brooklyn) 
+or, if you have the command-line tool `hub` ([described here](https://github.com/github/hub), or `sudo npm install -g hub`),
+by running this command:
+
+{% highlight bash %}
+hub fork; git submodule foreach 'hub fork'
+{% endhighlight %}
+
+The [Get the Code: Basics](index.html) page described how to retrieve the upstream repos, 
+but it gave those remotes the name `origin`.
+When using forks, `upstream` is a more accurate name. 
+You can rename the origin remotes with:
+
+{% highlight bash %}
+git remote rename origin upstream; git submodule foreach 'git remote rename origin upstream'
+{% endhighlight %}
+
+You'll now likely want to add the remote `origin` for your fork:
+
+{% highlight bash %}
+if [ -z "$GITHUB_ID" ] ; then echo -n "Enter your GitHub ID id: " ; read GITHUB_ID ; fi
+git remote add origin git@github.com:${GITHUB_ID}/apache
```

Comment:
Why not simply do `git remote rename ahgittin origin`?

If you prefer to keep this, then I guess it should be `/brooklyn` on the end instead of `/apache`.